src/collectd.conf.pod

```diff
@@ -6113,10 +6114,11 @@ Available configuration options:
 
 =over 4
 
-=item B<Host> I<IP-address>
+=item B<Host> I<IP-address> [I<Alias>]
```

Comment:
I'd prefer to have a config along the lines of:

```
<Host "app-prod-01-customer01.example.com">
  Alias "customer01.example.com"
</Host>
```

Have you checked the manpage if other plugins already use the option name `Alias`? If other plugins do something similar, we should try to use the same terminology if possible.